cfgov/unprocessed/css/molecules/inset.less

```diff
@@ -19,32 +26,102 @@
           .o-full-width-text
             .m-full-width-text
             .m-inset
-              .m-quote
-              .m-related-links
-              .m-call-to-action
-            .m-full-width-text
+            .m-inset__bottom-rule
+            .m-inset__left
+            .m-inset__portrait
+            .m-inset__150
+            .m-inset__300
+                .header-slug
+                    .header-slug_inner
+                .m-inset_image-container
+                    div.m-inset_image__cover
+                    div.m-inset_image__decorative
+                    img.m-inset_image
+                .m-inset_text
+                .m-quote
+                .m-related-links
+                .m-call-to-action
+                .m-image-inset
+                .m-full-width-text
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2678/commits/74c81849f5ecd2cb7d3e9cc07c71d8dc53489f31.